draft-irtf-cfrg-hash-to-curve.md

```diff
@@ -2159,7 +2158,8 @@ Output: (v, w), a point on E.
 The mappings of {{mappings}} always output a point on the elliptic curve,
 i.e., a point in a group of order h * r ({{bg-curves}}). Obtaining a point in G
 may require a final operation commonly called "clearing the cofactor," which
-takes as input any point on the curve.
+takes as input any point on the curve and produces as output a point in the
+prime-order (sub)group of the curve.
```

Comment:
"The prime-order (sub)group of the curve" reads like there's only one, but that might not be true. Would it make sense to make an explicit reference to G?

```suggestion
prime-order (sub)group G ({{bg-curves}}).
```